awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -21,6 +22,14 @@ const VariablesInput = styled(_VariablesInput)`
   }
 `;
 
+const StatusDetailValue = styled.div`
+  align-items: center;
+  display: inline-flex;
+  > div {
```

Comment:
The generic `div` selector here is likely to target unwanted other stuff that might get added later on. Can you either put a class on the div in question or use styled components to apply this margin directly to it? (I think this `div` targets the `StatusIcon`?)

Reply:
Good point - I'll push up a commit to add a class to the StatusIcon wrapper 👍